### DISCUSSION TITLE
BladeBit 3.0 Compressed plots

### DISCUSSION THREAD

**Discussion by majekqwert** [Chia-Network/chia-blockchain#16155] [Support]
It only work with 256gb of ram?
Can't create with 128GB + SSD? 
What about creating compressed plots with the CPU?

**Comment by shaneo257** [comment]
the latest version in 2.0.0 requires 256gb of ram
lower ram requirements + SSD is in the works and will be released very soon

**Reply by majekqwert** [reply]
Ok, so we are waiting for version 3.1. I have another question, maybe someone tested. With 500 TB for this i7-11700k with 128 GB of ram, what maximum compression level can I use for the processor to cope, especially assuming that in June 2024 the filter will be reduced.

**Reply by BrandtH22** [reply]
You can use the simulator to determine the maximum plots you can create at a given compression level:
https://www.youtube.com/watch?v=cZfptl66TLE
